Spec/Tests/AuthTests.swift

```diff
@@ -3045,19 +2992,8 @@ class AuthTests: XCTestCase {
             }
         }
 
-        let encoder = ARTJsonLikeEncoder()
-        encoder.delegate = ARTJsonEncoder()
-        let encodedTokenRequest: Data
-        do {
-            encodedTokenRequest = try encoder.encode(tokenRequest)
-        } catch {
-            fail("Encode failure: \(error)")
-            return
-        }
-        guard let tokenRequestJSON = String(data: encodedTokenRequest, encoding: .utf8) else {
-            XCTFail("JSON Token Request is empty")
-            return
-        }
```

Comment:
I approved this PR before but I have doubts looking at this?
1. it should be changed in a separate PR
2. don't we need to check if we can encode `tokenRequest` anymore?

Reply:
> don't we need to check if we can encode tokenRequest anymore?
Could you elaborate on this? I mean maybe you've seen changes in the docs related to this or some other hints.

Reply:
With this [comment](https://github.com/ably/ably-cocoa/pull/1258#discussion_r784825822) in mind can we agree that collapsing these lines into `XCTUnwrap` here can go into this PR as well?

Reply:
My bad 🙈 it's correct, I found it here @maratal